docs/core/docker/introduction.md

```diff
@@ -7,41 +7,62 @@ ms.custom: "mvc"
 
-There are a few concepts you should be familiar with. The Docker client has a CLI that you can use to manage images and containers. As previously stated, you should take the time to read through the [Docker overview](https://docs.docker.com/engine/docker-overview/) documentation.
+Official .NET Docker images are published to the [Microsoft Artifact Registry](https://mcr.microsoft.com/) and are discoverable on the [Docker Hub](https://hub.docker.com/_/microsoft-dotnet/). There are [runtime images](https://mcr.microsoft.com/product/dotnet/aspnet/) for production and [SDK images](https://mcr.microsoft.com/product/dotnet/sdk/) for building your code, for Linux (Alpine, Debian, Ubuntu) and Windows. For more information, see [.NET container images](container-images.md).
 
-### Images
+.NET images are regularly updated, whevever a new .NET patch is published or when an operating system base image is updated.
 
-An image is an ordered collection of filesystem changes that form the basis of a container. The image doesn't have a state and is read-only. An image is commonly based on another image, but with some customization. For example, when you create a new image for your application, you would base it on an existing image that already contains the .NET runtime.
+[Chiseled container images](https://devblogs.microsoft.com/dotnet/announcing-dotnet-chiseled-containers/) are Ubuntu container images with only the few components required by the .NET runtime. These images are ~100MB smaller than the regular Ubuntu images and have fewer [CVEs](https://www.cve.org/) since they have fewer components. In particular, they do not container a shell or package manager, which significantly improves their security profile. They also include a [non-root user](https://devblogs.microsoft.com/dotnet/securing-containers-with-rootless/) and are configured with that user enabled.
 
-Because containers are created from images, images have a set of run parameters (such as a starting executable) that run when the container starts.
+## Building container images
 
-### Containers
+You can build a container image with a **Dockerfile** or rely on the [.NET SDK to produce an image](https://devblogs.microsoft.com/dotnet/announcing-builtin-container-support-for-the-dotnet-sdk/). For samples on building images, see [dotnet/dotnet-docker](https://github.com/dotnet/dotnet-docker/blob/main/samples/README.md) and [dotnet/sdk-container-builds](https://github.com/dotnet/sdk-container-builds).
 
-A container is a runnable instance of an image. As you build your image, you deploy your application and dependencies. Then, multiple containers can be instantiated, each isolated from one another. Each container instance has its own filesystem, memory, and network interface.
+The following example demonstrates building and running a container image in a few quick steps.
 
-### Registries
+```bash
+$ dotnet new webapp -o webapp
+$ cd webapp/
+$ dotnet publish -p:PublishProfile=DefaultContainer
```

Comment:
This will only work with .NET 8. I know that you can set the context of the docs to be for a specific .NET version. I don't know how that is configured wrt these markdown files.

Reply:
This'll work for web projects out of the box from 7.0.300 onwards, but that's a nuance that probably isn't worth noting here :) In addition, we should possibly move away from `-p:PublishProfile=DefaultContainer` to `/t:PublishContainer` instead - folks seem to understand MSBuild Targets a bit more than the Publish Profile concept, and the target-form works better with solution-level publishing.

Reply:
Right, but what about the people using .NET 6? This needs to be excluded from .NET 6 context or clarified in the text.